vendor/k8s.io/kubernetes/pkg/credentialprovider/config.go

```diff
@@ -60,7 +60,7 @@ var (
 	preferredPathLock sync.Mutex
 	preferredPath     = ""
 	workingDirPath    = ""
-	homeDirPath, _    = os.UserHomeDir()
```

Comment:
Please don't modify the vendor files directly. If you need to modify them, you need raise PR in their repo first, then upgrade the vendor.

Reply:
get it. thank u very much.